Brokerages/GDAX/GDAXBrokerage.cs

```diff
@@ -35,6 +35,8 @@ public partial class GDAXBrokerage : BaseWebsocketsBrokerage
     {
         private const int MaxDataPointsPerHistoricalRequest = 300;
 
+        private static readonly HashSet<string> FiatCurrencies = new List<string> { "EUR", "GBP", "USD" }.ToHashSet();
```

Comment:
nit - could use defined `Currencies`?

Reply:
These are the only three currencies allowed for deposits to Coinbase Pro:
https://help.coinbase.com/en/pro/managing-my-account